src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/PersonOversiktStatusRepository.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.syfo.personstatus.infrastructure.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.personstatus.application.IPersonOversiktStatusRepository
+
+class PersonOversiktStatusRepository(private val database: DatabaseInterface) : IPersonOversiktStatusRepository {
+
+    override fun updateArbeidsuforhetVurderingStatus(personIdent: PersonIdent, isAktivVurdering: Boolean): Int {
+        database.connection.use { connection ->
+            connection.prepareStatement(UPDATE_ARBEIDSUFORHET_VURDERING_STATUS).use {
+                it.setBoolean(1, isAktivVurdering)
+                it.setString(2, personIdent.value)
+                val rowsUpdated = it.executeUpdate()
```

Comment:
Går an å gjøre dette (sånn at man ikke trenger egen linje med return)

it.executeUpdate().also {
    connection.commit()
}